ui/src/components/config_section/ConfigSectionTitle.js

```diff
@@ -5,12 +5,13 @@ export const ConfigSectionTitle = ({ title, iconType }) => (
   <EuiTitle size="s">
     <span>
       <EuiTextColor color="success">
-        <span>
-          {!!iconType && (
-            <EuiIcon className="eui-alignBaseline" type={iconType} size="m" />
-          )}
-          &nbsp;{title}
-        </span>
+        <EuiSpacer size="s" />
```

Comment:
I basically just added another `EuiSpacer` at the top to realign the accordion title and icon.

Before:
![Screenshot 2024-07-16 at 12 34 18](https://github.com/user-attachments/assets/8af73b79-451e-448b-8ab8-33df9aa9d5d8)

After:
![Screenshot 2024-07-16 at 12 41 52](https://github.com/user-attachments/assets/37fc9cc2-581f-4f91-9bbb-7381322c4cce)